windows/ReactNativeCameraCPP/ReactCameraViewManager.cpp

```diff
@@ -0,0 +1,156 @@
+#include "pch.h"
+#include "ReactCameraViewManager.h"
+#include "ReactCameraConstants.h"
+
+#include <iomanip>
+
+namespace winrt {
+    using namespace Windows::UI::Xaml;
+    using namespace Windows::UI::Xaml::Media;
+    using namespace Windows::UI::Xaml::Controls;
+    using namespace Windows::Devices::Enumeration;
+    using namespace Windows::Foundation;
+    using namespace Windows::Foundation::Collections;
+    using namespace Windows::Media::MediaProperties;
+    using namespace Windows::Storage;
+    using namespace Windows::Storage::Streams;
+    using namespace Microsoft::ReactNative;
+} //namespace winrt
+
+
+namespace winrt::ReactNativeCameraCPP::implementation {
+    // static vector to contain all currently active camera view instances
+    // This is a temporary workaround to map the view tag to view instance, since
+    // Method call from ReactCameraModule constains view tag as parameter and we need
+    // to forward the call to CameraView instance.
+    std::vector<winrt::com_ptr<ReactNativeCameraCPP::ReactCameraView>> ReactCameraViewManager::m_cameraViewInstances;
+

```

Comment:
is it even possible to have more than one? if i understand correctly you can only use the camera in one view in winrt

Reply:
You can have multiple views in your RN app, for example displaying both front and back cameras at same time. Hence in that situation, we have will two ReactCameraView instances, each is backed by a seperate CaptureElement.

Since we only have one camera module instance, and there are method calls on the module instance to deal with camera view object (using a viewTag parameter). We need a static vector for us to find the view object from the view tage, eventually(not yet supported) Native module interfaces should be able to get view instance from ReactContext object, and we can remove this vector and FindCamera logic in this native module.